### ISSUE TITLE
Editable Themes

### ISSUE THREAD

**Issue by Stefanoko** [nuttyartist/notes#599]
It were great if the provided themes were customizable.
Sepia comes close to my personal preference, except for the faded font color.

**Comment by zjeffer** [comment]
You can change most colors in the [src/styles](https://github.com/nuttyartist/notes/tree/master/src/styles) folder, and then compile the application yourself. Some colors are sadly hardcoded.

But maybe one day we will make it possible to do change this in the application itself, without recompiling, thanks for your suggestion.

**Comment by nuttyartist** [comment]
You're right about the text font color not having enough contrast in Sepia, I'll fix this in the next PR.

Regarding options for more theme colors, I strive for simplicity and minimum number of options so people can focus on their content. Maybe we'll create an advanced window settings where people could customize things more thourgely in the future, but it won't come anytime soon. I'll keep this issue open tho to see demand.

**Comment by Stefanoko** [comment]
Thank you both.
I've never compiled anything before, this option might be over my head I am afraid, but thanks for the hint.

Simplicity and minimum number of options — I fully agree.
A better contrast in the Sepia will do the job for me. Looking forward to the next release.
